src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -80,15 +88,15 @@
         WHERE
           ((strategy_mutator_plugin = 0) OR (strategy_mutator_plugin IS NULL)) AND
           /* Query results from the past 30 days. Change as needed. */
-          DATE_DIFF(cast(current_timestamp() AS DATE), cast(_PARTITIONTIME AS DATE), DAY) < 31
+          DATE_DIFF(cast(current_timestamp() AS DATE), cast(_PARTITIONTIME AS DATE), DAY) < 6
         )
       WHERE
         /* Filter for unstable targets. */
         fuzzer_stddev < 150)
     GROUP BY
       strategy))
 ORDER BY
-  bandit_weight DESC
+  bandit_weight_medium_temperature DESC
 """
```

Comment:
Query will group by 'strategy_selection_method' once we have one day of experimental data from the initial distribution.

Reply:
Please leave a TODO comment to make this obvious from the code.